linode/databasepostgresql/framework_datasource_schema.go

```diff
@@ -11,6 +11,7 @@ import (
 
 var frameworkDatasourceSchema = schema.Schema{
 	Attributes: map[string]schema.Attribute{
+		// DEPRECATED: Use ID instead
```

Comment:
Can we move the note to the description of the attribute?

Reply:
Done!